[2025-01-28T06:06:02.869+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-28T06:06:02.879+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.transform_load_weather_data manual__2025-01-28T06:03:35.704051+00:00 [queued]>
[2025-01-28T06:06:02.884+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.transform_load_weather_data manual__2025-01-28T06:03:35.704051+00:00 [queued]>
[2025-01-28T06:06:02.884+0000] {taskinstance.py:2866} INFO - Starting attempt 2 of 3
[2025-01-28T06:06:02.901+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): transform_load_weather_data> on 2025-01-28 06:03:35.704051+00:00
[2025-01-28T06:06:02.905+0000] {standard_task_runner.py:72} INFO - Started process 2635 to run task
[2025-01-28T06:06:02.911+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'weather_dag', 'transform_load_weather_data', 'manual__2025-01-28T06:03:35.704051+00:00', '--job-id', '17', '--raw', '--subdir', '/home/ubuntu/airflow/dags/weather_dag.py', '--cfg-path', '/tmp/tmplnz6feud']
[2025-01-28T06:06:02.912+0000] {standard_task_runner.py:105} INFO - Job 17: Subtask transform_load_weather_data
[2025-01-28T06:06:02.948+0000] {task_command.py:467} INFO - Running <TaskInstance: weather_dag.transform_load_weather_data manual__2025-01-28T06:03:35.704051+00:00 [running]> on host ip-172-31-38-205.ap-southeast-2.compute.internal
[2025-01-28T06:06:03.139+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='trinhminhquan2810@gmail.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='weather_dag' AIRFLOW_CTX_TASK_ID='transform_load_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2025-01-28T06:03:35.704051+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-28T06:03:35.704051+00:00'
[2025-01-28T06:06:03.139+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-01-28T06:06:03.140+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-01-28T06:06:03.140+0000] {logging_mixin.py:190} INFO - Current task name:transform_load_weather_data state:running start_date:2025-01-28 06:06:02.879532+00:00
[2025-01-28T06:06:03.140+0000] {logging_mixin.py:190} INFO - Dag name:weather_dag and current dag run status:running
[2025-01-28T06:06:03.140+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-28T06:06:03.150+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-28T06:06:03.154+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-28T06:06:03.154+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=transform_load_weather_data, run_id=manual__2025-01-28T06:03:35.704051+00:00, execution_date=20250128T060335, start_date=20250128T060602, end_date=20250128T060603
[2025-01-28T06:06:03.168+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-01-28T06:06:03.168+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-01-28T06:06:03.168+0000] {logging_mixin.py:190} INFO - Dag name:weather_dag queued_at:2025-01-28 06:03:35.766221+00:00
[2025-01-28T06:06:03.169+0000] {logging_mixin.py:190} INFO - Task hostname:ip-172-31-38-205.ap-southeast-2.compute.internal operator:PythonOperator
[2025-01-28T06:06:03.209+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-28T06:06:03.219+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-28T06:06:03.220+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
